evals/roles/gitea/tasks/monitoring.yaml

```
@@ -0,0 +1,27 @@
+---
+- name: Check {{ gitea_namespace }} namespace exists
+  shell: oc get namespace {{ gitea_namespace }}
+  register: gitea_namespace_check
+  failed_when: false
```

Can you add an specific 'not found error" such as `gitea_namespace_check.stderr != '' and 'not found' not in gitea_namespace_check.stderr` - it could fail in case of an unexpected issue shows up  such as master is unreachable which we may want to throw an error